totest-manager.py

```diff
@@ -566,7 +566,7 @@ class ToTestFactoryzSystems(ToTestBase):
     main_products = ['_product:openSUSE-dvd5-dvd-s390x',
```

Comment:
Actually, when we switch to the new prod builder, the DVD ones also are produced by it - there can;t be a mix between both builders.

Please adjust those as well.